portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,77 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Adetowo Ademiluyi's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <body vlink="grey">
+    <h1 id="myHeader" font-size: 50px>ADE's PORTFOLIO!</h1>
+    <div class="content">
+      <div class = "overlay">
+        <div class = "menu">
+          <ul>
+            <li><a href = "index.html">Home</a></li>
+            <li><a href = "aboutME.html">About Me!</li>
+            <li><a href = "mailto:aaademiluyi99@gmail.com">Contact</li>
+            <li><a href = "projects.html">Projects</li>
+            <li><a href = "images.html" class = "gallery"><span>Gallery</span>
+            </a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class = "app-quote">
+        <div class = "app-text">
+          <h1> The future belongs to <br>
+               those who believe in the <br>beauty <br>
+               of thier dreams <h4> - Eleanor Roosevelt</h4>
+          </h1>
+        </div>
+      </div>
+
+      <div class = "image_block">
+             <img src="new_images/lagos-6.jpg" width = "100" height = "100"/>
```

Comment:
The folder's name "new_images" could be changed to "images" or "gallery_images". It is more clear about the contents that are included by the folder.

Reply:
i will fix it!